### PR TITLE
Update to 3.5.1

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
Fixes https://github.com/conda-forge/pyepics-feedstock/pull/38

**Review by tacaswell — DISMISSED** [review]

**Comment by tacaswell** [comment]
This is going to be the same concern for both this PR and #42 .

We have gone through several rounds of patching pyepics on CF to not bundle the pre-built binaries that are improperly checked into the source repo and instead the pyepics conda package correctly use libca and friends from the epics conda package. 

From the patch that is removed it looks like we have been in a broken state for a while (it looks like it was trimming to "only" ship the binaries related to the platform we are on).  I think that without this patch we are back to shipping _all_ of the binaries.

I think we need to pick up a patch that purge those binaries again.

**Review by tacaswell — CHANGES_REQUESTED** [review]
This is shipping pre-built binaries that will be used instead of the libca from the epics-base package.

File: recipe/build.sh [hunk 1]
```diff
@@ -9,3 +9,5 @@ mkdir -p $PREFIX/etc/conda/deactivate.d
 cat <<EOF > $PREFIX/etc/conda/deactivate.d/pyepics_deactivate.sh
 unset PYEPICS_LIBCA
 EOF
+
+rm -r epics/clibs
```
Comment (tacaswell) [inline recipe/build.sh:13]: should we do this before we install?

Comment (jan-janssen) [reply recipe/build.sh]: I agree

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@tacaswell I removed the binaries by patching the `setup.py` file. Is that fine for you? Then I can also apply it for the `3.5.2` release.

**Comment by jan-janssen** [comment]
@tacaswell Can you take another look at this pull request?

**Pull request merged**